src/STEMMUS_SCOPE_exe.m

```diff
@@ -24,8 +24,12 @@ function STEMMUS_SCOPE_exe(config_file, runMode)
                        'KT', ... % Index of current time step
                        'SiteProperties', ... % Site properties (e.g. lat, lon)
                        'fluxes', ... % Atmospheric fluxes
-                       'TT' ... % Soil temperature over depth
-                      }; %#ok
+                       'TT', ... % Soil temperature over depth

```

Comment:
The variable `GroundwaterSettings` should be also added to this list.

Reply:
Can you also remove 'ModelSettings' from this list? Currently, it isnot possible to set/get this variable through bmi.